events/message.js

```diff
@@ -1,9 +1,29 @@
 const Event = require(`${process.cwd()}/base/Event.js`);
 const monitor = require(`${process.cwd()}/monitors/monitor.js`);
-const Social = require("../base/Social.js");
+const Social = require(`${process.cwd()}/base/Social.js`);
+const { Permissions } = require("discord.js");
 
 module.exports = class extends Event {
 
+  constructor(...args) {
+    super(...args);
+    this.impliedPermissions = new Permissions([
+      "VIEW_CHANNEL",
+      "SEND_MESSAGES",
+      "SEND_TTS_MESSAGES",
```

Comment:
It does not, and will not require `SEND_TTS_MESSAGES`

Reply:
It's implied permissions not what it requires.

Reply:
But still, it doesn't need that like at all

Reply:
I know it doesn't, But it is not permissions needed it is implied permissions..........